bin/python/ripple/build/module/Default.py

```diff
@@ -0,0 +1,215 @@
+from __future__ import (
+    absolute_import, division, print_function, unicode_literals)
+
+import os
+
+from beast.build.Module import Module
+
+from beast.build.Function import Env, compose, for_tags, not_tags
+
+Append = Env.Append
+Prepend = Env.Prepend
+
+
+MODULE = Module(
+    files=compose(
+        for_tags(
+            'debug',
+            Append(CPPDEFINES=['DEBUG', '_DEBUG']),
+        ),
+
```

Comment:
I'm _very_ open to another name - I've gone through various incarnations of it over development!

A tag is a property of a build like `gcc`, `debug`, `unity` or `osx`.

The full sentence would be:  "For builds with tags [list of tags], do [this thing]" - `for_tags`.

Earlier incarnations were `Tags`, `with_tags`, `do_tags`


Tags are documented [here](https://github.com/rec/rippled/blob/25928e124567d9c01eeccee4ac7e5cd137ec0878/src/beast/python/beast/build/TagSet.py#L12) though I could probably revisit that.